tcms/testcases/migrations/0018_add_testcase_duration_fields.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.1.6 on 2021-02-16 09:40
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('testcases', '0017_rename_related_names'),

```

Comment:
See lint/black failures.  This code base uses `black` to reformat the code automatically. Type `black <filename>` on your machine and it should format the code for you.